feature-tests/tests/resources/step-definitions/invoke-api-gateway-happy-path.step.ts

```diff
@@ -0,0 +1,37 @@
+import { defineFeature, loadFeature } from 'jest-cucumber';
+import { generateRandomTestUserId } from '../../../utils/generate-random-test-user-id';
+import { sendSQSEvent } from '../../../utils/send-sqs-message';
+import { invokeApiGateWayLambda } from '../../../utils/invoke-apigateway-lambda';
+
+const feature = loadFeature('tests/resources/features/vcsGET/invokeApiGateway-HappyPath.feature');
```

Comment:
```suggestion
const feature = loadFeature('tests/resources/features/statusGet/invokeApiGateway-HappyPath.feature');
```
Or something that relates to ais.